nautobot/extras/templates/extras/jobresult.html

```diff
@@ -80,86 +77,104 @@
-                    </div>
-                </div>
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <strong>File Output(s)</strong>
-                    </div>
-                    <div class="panel-body">
-                        <ul>
-                            {% for file_proxy in result.files.all %}
-                                {% if file_proxy.file %}
-                                    <li>
-                                        <a
-                                        {% if settings.JOB_FILE_IO_STORAGE == "db_file_storage.storage.DatabaseFileStorage" %}
-                                            href="{% url "db_file_storage.download_file" %}?name={{ file_proxy.file }}"
-                                        {% else %}
-                                            href="{{ file_proxy.file.url }}"
-                                        {% endif %}
-                                            download="{{ file_proxy.name }}">
-                                            {{ file_proxy.name }}
-                                        </a>
-                                    </li>
-                                {% endif %}
-                            {% endfor %}
-                        </ul>
-                    </div>
-                </div>
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <strong>Traceback</strong>
-                    </div>
-                    <div class="panel-body">
-                        {% include 'extras/inc/json_data.html' with data=result.traceback format="python" %}
-                    </div>
-                </div>
-            </div>
+{% block advanced_content_left_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Job Positional Arguments</strong>
+        </div>
+        <div class="panel-body">
+            {% include 'extras/inc/json_data.html' with data=result.task_args format="json" %}
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Job Keyword Arguments</strong>
+        </div>
+        <div class="panel-body">
+            {% include 'extras/inc/json_data.html' with data=result.task_kwargs format="json" %}
         </div>
     </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Job Celery Keyword Arguments</strong>
+        </div>
+        <div class="panel-body">
+            {% include 'extras/inc/json_data.html' with data=result.celery_kwargs format="json" %}
+        </div>
+    </div>
+{% endblock advanced_content_left_page %}
+{% block advanced_content_right_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Worker</strong>
```

Comment:
Now I'm *really* nitpicking, but I don't know that I like this being the first / most prominent panel in the right column. Logically it makes sense (inputs on the left, then worker info, then outputs on the right) but it just seems like info that's less relevant to most users.

**Maybe** put the Worker panel at the bottom of the left column instead, even though that results in some rather unbalanced columns?

Reply:
The columns could also be potentialy be unbalanced due to the Custom Fields.

Reply:
After discussing in Parking Lot, we are going to keep Worker Data on the right, but remove File Output, as that is already in the main tab. I will update the screenshot with better examples.